### ISSUE TITLE
More weather icons?

### ISSUE THREAD

**Issue by Feichtmeier** [ubuntu/yaru_icons.dart#133]
While thinking about the weather app, I wonder two things:

1) could we have more weather icons, which also include appropriate icons for the night icons and moon phases
![grafik](https://user-images.githubusercontent.com/15329494/220637059-49a23d32-597b-45b1-96d8-e0256359e585.png)

2) What would be the best way to get weather icons that only color certain parts of the icon with "blue" for the moon, "yellow"/"orange" for the sun and "blue" for rain drops?

**Comment by Jupi007** [comment]
I agree, that would be nice to have more weather icons, and I'll work on 👍

On the colour point, we cannot color a specific part of an `Icon`.
To archive this, I can work on some `CustomPainter` based icons. It would be relatively simple to implement 🙂

**Comment by Feichtmeier** [comment]
Or I look into how you did it and do it myself?
I feel bad for requesting all those icons all the time

**Comment by Jupi007** [comment]
> Or I look into how you did it and do it myself?
> I feel bad for requesting all those icons all the time

No, please don't feel bad for this.
It is the only Flutter field of competence I'm relatively mastering 😅
And it is always a pleasure for me to be helpful!

**Comment by Jupi007** [comment]
Sounds very promising! 😀

**Comment by Feichtmeier** [comment]
*soft reminder* :rofl:

**Comment by Jupi007** [comment]
> *soft reminder* :rofl: 

Ahah, I'll work on it after the heart 😁

**Comment by Jupi007** [comment]
@Feichtmeier Here are all our weather related icons:

![image](https://github.com/ubuntu/yaru_icons.dart/assets/36476595/52f942da-6944-458b-a50a-77dd9b1e96bd)

To be in sync with openweather icons, we just miss a night version of `showers`.
Is there anything else you need?

After what I will work on colorable version of those icons.

**Comment by Feichtmeier** [comment]
Yes, it would be nice to have versions that are full color like in the opening post. But ofc this is much work and depening on how the app ends up in design not even 100% needed

**Comment by Feichtmeier** [comment]
Evenually I can get the result with stack + positioned and color certain parts

**Comment by Jupi007** [comment]
> Evenually I can get the result with stack + positioned and color certain parts

Please don't, use a stack will be ugly.
I started to work on the recolorable icons :)

**Comment by Feichtmeier** [comment]
Cool thank you!
